sdk/storage/Azure.Storage.Blobs/src/BlobContainerClient.cs

```diff
@@ -2084,10 +2084,9 @@ private void SetNameFieldsIfNull()
 
         #region UploadBlob
         /// <summary>
-        /// The <see cref="UploadBlob"/> operation creates a new block
-        /// blob or updates the content of an existing block blob in this
-        /// container.  Updating an existing block blob overwrites any existing
-        /// metadata on the blob.
+        /// The <see cref="UploadBlobAsync"/> operation creates a new block
+        /// blob.  A <see cref="RequestFailedException"/> will be thrown
+        /// if the blob already exists.
```

Comment:
Should we move this to the `<remarks>` section where we talk about exceptions that get thrown?

Reply:
Moved.